tools/accuracy_checker/configs/openpose-cmu-tf.yml

```diff
@@ -0,0 +1,54 @@
+models:
+  - name: openpose-cmu-tf
+
+    launchers:
+      - framework: tf
+        model: public/openpose-cmu-tf/opt_graph.pb
+        adapter:
+          type: human_pose_estimation
+          part_affinity_fields_out: Mconv7_stage6_L1/Conv2D
+          keypoints_heatmap_out: Mconv7_stage6_L2/Conv2D
+
+    datasets:
```

Comment:
if original framework and IR does not have difference in preprocessing, no need to specify source framework as separated model, you can have only one model specification with 3 launchers (tf, dlsdk with FP32 tag and dlsdk with FP16 tag)